src/web/views/children-dob.njk

```diff
@@ -13,7 +13,7 @@
     }
   }) %}
 
-  {%  for i in range(0, children.count) %}
+  {%  for i in range(0, children.count + 1) %}
```

Comment:
Won't this always add one more child than is in the session?

Reply:
Yes - it was working because on submit the count was not being incremented for the first child added, but that is wrong.